types/Keyguard.d.ts

```diff
@@ -32,12 +32,38 @@ type AccountRecord = AccountInfo & {
     encryptedKeyPair: Uint8Array
 }
 
+type EncryptionKeyParams = {
+    kdf: string
+    iterations: number
+    keySize: number
+}
```

Comment:
Can be used as `KeyguardRequest.EncryptionKeyParams` without copying it from `PublicRequest.ts`.

```suggestion
type EncryptionKeyParams = KeyguardRequest.EncryptionKeyParams;
```

Reply:
Resolved in #524